google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -51,7 +51,7 @@
   private final double consistency;
   private int port;
```

Comment:
you don't need `port` here, it will be inherited from the parent class

Reply:
done